tests/abi/tools/revng-ensure-rft-equivalence/Main.cpp

```diff
@@ -0,0 +1,119 @@
+/// \file Main.cpp
+/// \brief
```

Comment:
Can you describe *in detail* what this tool does?
> This tool takes two models, looks for all the `RawFunctionType`s with the same ID...
> Using `renvg model diff` would not work since...